workbench-core/authorization/src/models/getUsersFromGroup.ts

```diff
@@ -3,15 +3,22 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
+import { z } from 'zod';
+
 /**
  * Request object for DynamicPermissionsPlugin's getUsersFromGroup
  */
-export interface GetUsersFromGroupRequest {
```

Comment:
see previous comment about leaving this file alone

Reply:
Fixed !